tests/lib/rules/all.js

```diff
@@ -76,15 +83,6 @@ ruleTester.run('underscore.isNaN', rules['is-nan'], {
   invalid: [{
     code: 'underscore.isNaN(NaN)',
     errors: ['Consider using the native Number.isNaN()']
-  }, {
-    code: "import isNaN from 'lodash/isNaN';",
-    errors: ["Import from 'lodash/isNaN' detected. Consider using the native Number.isNaN()"]
-  }, {
-    code: "import isNaN from 'lodash.isnan';",
-    errors: ["Import from 'lodash.isnan' detected. Consider using the native Number.isNaN()"]
-  }, {
-    code: "import { isNaN } from 'lodash';",
-    errors: ["Import { isNaN } from 'lodash' detected. Consider using the native Number.isNaN()"]
```

Comment:
these tested were for lodash but were under `underscore`.
Moved them up to the lodash test block and made it so it's now easier to add new test cases.